content/docs/command-reference/remote/modify.md

```diff
@@ -683,6 +687,78 @@ more information.
 
 <details>
 
+### Click for WebHDFS
+
+💡 WebHDFS serves as an alternative for using the same remote storage supported
+by HDFS. Read more about by expanding the WebHDFS section in
+[`dvc remote add`](/doc/command-reference/remote/add#supported-storage-types).
+
+- `url` - remote location:
+
+  ```dvc
+  $ dvc remote modify myremote url webhdfs://user@example.com/path/to/dir
+  ```
+
+- `user` - username to access the remote, can be empty in case of using `token`
+  or if using a `HdfsCLI` cfg file. May only be used when Hadoop security is
+  off. Defaults to current user as determined by `whoami`.
+
+  ```dvc
+  $ dvc remote modify --local myremote user myuser
+  ```
+
+- `token` - Hadoop delegation token for WebHDFS, can be empty in case of using
+  `user` or if using a `HdfsCLI` cfg file. May be used when Hadoop security is
+  on.
+
+  ```dvc
+  $ dvc remote modify --local myremote token mytoken
+  ```
+
+- `hdfscli_config` - path to a `HdfsCLI` cfg file. WebHDFS access depends on
+  `HdfsCLI`, which allows the usage of a configuration file by default located
+  in `~/.hdfscli.cfg`. In the file, multiple aliases can be set with their own
+  connection parameters, like `url` or `user`. If using a cfg file,
+  `webhdfs_alias` can be set to specify which alias to use.
```

Comment:
Could simplify a lot:
```suggestion
- `hdfscli_config` - path to a `HdfsCLI` cfg file, by default located
  in `~/.hdfscli.cfg`. If using a cfg file,
```

And ideally link somewhere instead of explaining what's inside .cfg files, maybe https://hdfscli.readthedocs.io/en/latest/quickstart.html? (`webhdfs_alias` referring back to this should be enough, I think).